docs/requirements.txt

```diff
@@ -1,7 +1,8 @@
 -f http://simple.crate.io/
 MarkupSafe==0.23
 Pygments==2.0.2
-Sphinx==1.3.1
-
-sphinx-autobuild==0.5.0
-sphinx_rtd_theme # always use latest version of the theme
+sphinx
```

Comment:
BTW, ``pip install sphinx`` is actually what the official docs say, so I think it doesn't matter.